src/Components/Artwork/SaveButton/SaveArtworkToListsButton.tsx

```diff
@@ -117,3 +119,51 @@ const labelByResultAction = {
   [ResultAction.SavedToDefaultList]: "Saved Artwork",
   [ResultAction.RemovedFromDefaultList]: "Removed Artwork",
 }
+
+interface SaveArtworkToListsButtonQueryRendererProps
+  extends Omit<SaveArtworkToListsButtonProps, "artwork"> {
+  id: string
+}
+
+export const SaveArtworkToListsButtonQueryRenderer: FC<SaveArtworkToListsButtonQueryRendererProps> = ({
+  id,
+  contextModule,
+}) => {
+  return (
+    <SystemQueryRenderer<SaveArtworkToListsButtonQuery>
+      lazyLoad
+      query={graphql`
+        query SaveArtworkToListsButtonQuery($id: String!) {
+          artwork(id: $id) {
+            ...SaveArtworkToListsButton_artwork
+          }
+        }
+      `}
+      placeholder={
+        // TODO: This should be a better placeholder, rather than reusing `SaveButtonBase`
```

Comment:
Bit of a hack, trying to reuse the main React component (which is typed with Relay data) as the placeholder